quiz.js

```diff
@@ -36,25 +52,63 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Ned Stark' }
+});
+
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Arya Stark' }
+});
+
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Sansa Stark' }
+});
+
+db.westeros.update({
+    name: 'Targaryen'
+}, {
+    $push: { people: 'Viserys Targaryen' }
+});
+
+db.westeros.update({
+    name: 'Targaryen'
+}, {
+    $push: { people: 'Daenerys Targaryen' }
+});
+
+db.westeros.update({
+    name: 'Jon Arryn'
+}, {
+    $push: { people: 'Jon Arryn' }
+});
 
+// your code ends here
```

Comment:
This is all correct. However, writing it all out like this might get kind of tedious if we needed to add a lot of people to each house. Fortunately, Mongo gives us a tool to make this simpler ; $each.
```
db.westeros.update({ name : 'Stark' }, { $push : { people : 
  { $each : [
    'Ned Stark',
    'Arya Stark',
    'Sansa Stark'
  ] }
} });
```